extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -146,6 +147,10 @@ export class SubscriptionService {
    */
   registerPlatform(serviceId, subscriptionPlatformFactory) {
     return this.initialize_().then(() => {
+      console.log('hi1');
+      if (this.viewer_.hasCapability('auth')) {
```

Comment:
Can we store this in constructor as a const var?

Reply:
Done